src/nonvisual/mousemanager.js

```diff
@@ -132,6 +132,7 @@ var MouseManager = Base.$extend({
         } else {
             this.$super(element);
         }
+        this.__dragStartButton = null; // used for state manager
```

Comment:
This should be declared and documented line 438

Reply:
Corrected